.github/workflows/publish_artifacts.yml

```diff
@@ -1,120 +0,0 @@
-name: Publish Artifacts
```

Comment:
Thanks @i-walker !! This file must exist. I'll add a new commit in a few minutes!

Reply:
Thanks @rachelcarmena 😄